pkg/session/session.go

```diff
@@ -220,7 +225,7 @@ func newClient(ctx context.Context, logger logr.Logger, sessionKey string, url *
 			_, err := methods.GetCurrentTime(ctx, tripper)
 			if err != nil {
 				logger.Error(err, "failed to keep alive govmomi client")
-				logger.Info("clearing the session", "key", sessionKey)
+				logger.Info("clearing the session", "user", url.User.Username())
```

Comment:
```suggestion
				logger.Info("clearing the session", "username", url.User.Username())
```

for consistency, also below

Reply:
Let's log server, datacenter and username

Just makes it easier to identify a session

Reply:
We also shouldn't log or print the sessionKey IMHO even if it's hashed.

Reply:
As discussed in the office hours. Logging server, datacenter and username is fine. We just shouldn't log the hashed password

Reply:
will update logging in next patch